terraform/modules/concourse_web/web_config.tf

```diff
@@ -37,9 +37,12 @@ locals {
       #CONCOURSE_OIDC_GROUPS_KEY      = "cognito:groups"
       #CONCOURSE_OIDC_USER_NAME_KEY   = "cognito:username"
 
-      CONCOURSE_DATADOG_AGENT_HOST = "localhost"
-      CONCOURSE_DATADOG_AGENT_PORT = 8125
-      CONCOURSE_DATADOG_PREFIX     = "concourse"
+      # WE DON'T HAVE A DATADOG IMPLIMENTATION?
```

Comment:
I think I'd like these to commented out lines removed now; we're unlikely to ever implement datadog. This can wait until we're changing this file again for Cognito though to save another PR CI cycle.